Project.toml

```diff
@@ -7,13 +7,15 @@ ArnoldiMethod = "ec485272-7323-5ecc-a04f-4719b315124d"
 Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
 DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
```

Comment:
Why did you add this dependency? It doesn't seem needed and would make the package much heavier to load
```suggestion
```

Reply:
Sorry, I think some undesired dependencies have been added because of my dubious workflow.

Reply:
The [documentation of Pkg.jl](https://pkgdocs.julialang.org/v1/getting-started/) is a good place to start improving said workflow ;)